dashboard/app/models/concerns/user/password_validations.rb

```diff
@@ -0,0 +1,64 @@
+# This concern adds password validation logic to the User model.
+# It includes default validations for presence, length, and confirmation,
+# and applies stricter password rules for certain countries based on feature flags.
+
+module User::PasswordValidations
+  extend ActiveSupport::Concern
+
+  # Password Constants
+  PASSWORD_MAX_LENGTH = 128
+  PASSWORD_MIN_LENGTH = 6
+  PASSWORD_STRICT_MIN_LENGTH = 14
+  # Countries that require a 14 character password minimum
+  PASSWORD_STRICT_COUNTRIES = %w[AU NZ].freeze
+
+  included do
+    validates_presence_of     :password, if: :password_required?
+    validates_confirmation_of :password, if: :password_required?
+    validates_length_of       :password, minimum: :password_min_length, maximum: :password_max_length, allow_blank: true
+  end
+
+  class_methods do
+    def password_min_length(user_type, country_code)
+      if user_type == User::TYPE_TEACHER && PASSWORD_STRICT_COUNTRIES.include?(country_code) && strict_password_enabled?
+        PASSWORD_STRICT_MIN_LENGTH
+      else
+        PASSWORD_MIN_LENGTH
+      end
+    end
+
+    def strict_password_enabled?
```

Comment:
Nice, I like having this broken out into it's own method. Makes the conditional statement above more readable 👍